docs/advanced/bootloader.md

```diff
@@ -94,3 +98,18 @@ It produces the following ROM layout:
 ``` 
 
 For an example showing how to create an application that uses a bootloader, see the [mbed OS bootloader example](https://github.com/armmbed/mbed-os-example-bootloader-blinky) repository.
+
+## Unmanaged bootloader
+
+You want to have an unmanaged bootloader when your bootloaders requirements conflict with the requirements of the managed bootloader. You need an unmanaged bootloader when your bootloader does not come before your application in ROM or your application does not start immediately after your bootloader. Unlinke a managed bootloader, an unmananged bootloader does not automatically merge the bootloader image with the application image after building the application. We expect a user of an unmanaged bootloader build to construct there own set of scripts built atop the `mbed compile` primitive to perform bootloader and application merging.
+
+An unmanaged bootloader build is a method for controlling the link location of a program within mbed OS. There are two configuration options available for changing the link location: `target.mbed_app_start` and `target.mbed_app_size`. 
+
+### `target.mbed_app_start`
+
+
+The configuration option `target.mbed_app_start` sets the starting address of the linker script by defining the `MBED_APP_START` macro for the linker script. This configuration option may only be defined within the `target_overrides` section of an mbed application configuration and may not be defined for the meta-target `*`. When this configuration option is not defined, it defaults to the start of a target's ROM. This configuration option must be an address within ROM.
+
+### `target.mbed_app_size`
+
+The configuration option `target.mbed_app_size` defines the size of an application image in ROM by defining the `MBED_APP_SIZE` macro for the linker script. This configuration option may only be defined on a per-target basis defined within the `target_overrides` section of an mbed application configuration and may not be defined for the meta-target `*`. When this configuration option is not defined, it defaults to the  remaining ROM, which is calculated by subtracting the image's offset into ROM from the total size of ROM. Together with `target.mbed_app_start`, these configuration options define a continuous region of memory that an image may use. The tools verify that this region of memory is placed within ROM, but the tools do not perform any other checks for consistency or validity.
```

Comment:
- "may only be defined" and "may not be defined" and "is not defined" are passive voice. Please clarify who or what is doing the defining.
- Please delete the extra space between "the" and "remaining" in the third sentence.
- "is calculated" is passive voice. Please clarify who or what is doing the calculating.
- "is placed within ROM" is passive voice. Please clarify who or what is doing the placing.